src/utils/consts/board.ts

```diff
@@ -0,0 +1,9 @@
+const BOARD_SIZE = 8;
+const CELL_SIZE = 100;
```

Comment:
Here and in the next file I'd recommend mentioning units in the name of the consts like `CELL_SIZE_PX` or `OTHER_SIZE_PERCENT`
Later on you'll decide to change units and this value might become confusing

Reply:
Done